plugins/modules/zos_copy.py

```diff
@@ -778,7 +794,8 @@ def __init__(
         module,
         is_binary=False,
         executable=False,
-        backup_name=None
+        backup_name=None,
+        force_lock=False,
```

Comment:
I think it would be helpful if you added a description in the docstring for the `force_lock` argument, at least on this class.

Reply:
Agree, done